idtools/idmap.go

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
idmap package will be from containerd and will be throw away.
license to be for containerd.